koin-projects/koin-androidx-compose/src/main/java/org/koin/androidx/compose/KoinCompose.kt

```diff
@@ -0,0 +1,28 @@
+package org.koin.androidx.compose
+
+import androidx.compose.Composable
+import androidx.compose.remember
+import org.koin.core.context.KoinContextHandler
+import org.koin.core.parameter.ParametersDefinition
+import org.koin.core.qualifier.Qualifier
+
+/**
+ * @author Henrique Horbovyi
+ * */
+
+/**
+ * Inject lazily a given dependency for [Composable] functions
+ * @param qualifier
+ * @param parameters
+ *
+ * @return Lazy instance of type T
+ */
+
+@Composable
+inline fun <reified T> inject(
```

Comment:
Add perhaps @RequireOptIn here, to ask people to opt in. As we need to see if it's stable enough